services/QuillLMS/app/helpers/mobile_helper.rb

```diff
@@ -0,0 +1,19 @@
+module MobileHelper
+  def render_dash(is_mobile)
+    if !is_mobile
+      '<div class="light-bar"></div>'.html_safe
+    end
+  end
+
+  def render_content(device, type, url)
```

Comment:
What about `render_video_content` since the implementation references `video-section`.